css/style.css

```diff
@@ -401,30 +404,51 @@ header {
     transition: width 0.1s ease-in-out;
   }
 
-  .hero-section {
-    background-image: url("../img/image-geometry.svg");
-    background-repeat: no-repeat;
-    background-position: right;
-    background-size: 45%;
-    background-position-y: top;
-  }
-
   .hero-section > img {
     display: none;
   }
 
   .hero-content {
-    max-width: 50%;
+    display: flex;
+    justify-content: stretch;
+    align-items: stretch;    
   }
 
-  .hero-content > h1 {
+  .hero-left {
+    padding: 10% 20px 200px 8%;
+    background: url("../img/image_dots.svg");
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    background-size: 138px;
+    width: 50%;
+  }  
+  
+  .hero-right {
+    background: url("../img/image-geometry.svg");
+    background-repeat: no-repeat;
+    background-position: right;
+    background-size: cover;
+    width: 50%;
+  }
```

Comment:
Nicely done, **Wuletaw**! The two backgrounds are very well aligned and scaled to match the design :+1: 

I also like the use of the empty `div` for the image as also makes it clear in the HTML markup what the two parts of the hero are.

You could **make the two backgrounds stay in sync** when the page is scaled (aka make the hero more responsive) by using height-related percentages for scaling the images and aligning them to their common axis (just as you've already said/did in the uncommitted code you showed in the presentation)

Reply:
Thanks @razvanso,
I have applied the suggestion you gave about making the backgrounds to stay in sync when the viewport scales. As per your suggestion I've used percentages instead of pixels and it syncs with the height of the right hero container background.